perllib/FixMyStreet/App.pm

```diff
@@ -387,15 +396,16 @@ sub send_email_cron {
     $params->{_line_indent} = '';
     my $email = mySociety::Locale::in_gb_locale { mySociety::Email::construct_email($params) };
 
-    if ( FixMyStreet->test_mode ) {
-        my $sender = Email::Send->new({ mailer => 'Test' });
-        $sender->send( $email );
-        return 0;
-    } elsif (!$nomail) {
-        return mySociety::EmailUtil::send_email( $email, $env_from, @$env_to );
-    } else {
+    if ($nomail) {
         print $email;
         return 1; # Failure
+    } else {
+        my %model_args;
+        if (!FixMyStreet->test_mode && $env_from == FixMyStreet->config('CONTACT_EMAIL')) {
+            $model_args{mailer} = 'FixMyStreet::EmailSend::ContactEmail';
```

Comment:
If you're not running in test mode (as then you always want the mailer to be `Test`) if the from is the contact address, this sets the mailer to the one that has a sender envelope also of the contact address. It looks to be okay with a quick print debugging, where do you think it's overridden?